docs/technical/api-spec.yml

```diff
@@ -33,42 +32,108 @@ paths:
             format: uuid
             example: bb20ba03-5e52-4a61-bcc1-0ad538f1b448
           description: 'Uuid generated by the sender of a message for correlating async responses to a specific request. '
+        - name: X-Send-To
+          in: header
+          required: true
+          schema:
+            type: string
+            example: Classifier
+          description: Participant name to send the message to.
+        - name: X-Message-Id-Ref
+          in: header
+          schema:
+            type: string
+            format: uuid
+            example: 545226ad-0b8d-4ccb-b83f-6b3ad8d44ec8
+          description: Reference to the original X-Message-Id we are replying to.
       description: ''
       requestBody:
         content:
-          text/plain:
+          application/x.classifier.classification+json;version=1:
             schema:
-              type: string
-              description: Encoded payload
-          application/json:
-            schema:
-              title: ClassificationRequest
-              type: array
-              items:
-                type: string
-                example: Greetings from Estonia!
+              title: Classification
+              type: object
+              properties:
+                message:
+                  type: string
+                  example: Greetings from Estonia!
+                classification:
+                  type: string
+                  example: NLib
+                  readOnly: true
+                context:
+                  type: object
+              required:
+                - message
+                - classification
         description: ''
     parameters: []
 components:
   schemas:
-    ClassificationRequest:
-      title: ClassificationRequest
-      type: array
-      items:
-        type: string
-        example: Greetings from Estonia!
-    ClassificationResponse:
-      title: ClassificationResponse
+    Classification:
+      title: Classification
+      x-stoplight:
```

Comment:
good catch, im actually exporting with "remove stopligh tags", so if any come through it is usually because I have modified the underlying structure manyally, will fix